### PR TITLE
academic_procedure

### PR DESCRIPTION
**Pull request by achintMI**
Add Course, Drop Course, Registration, Pre-Registration, Branch Change Request, Verify Course, Admin Drop Course, Process Branch Request
These are completed with Ajax and will contain some minor bugs

### CODE REVIEW

**Comment by Samy-33** [comment]
@achintMI please re-pull and push again. There is a conflict, in sidenavbar. New code was added.

**Comment by achintMI** [comment]
Please check ASAP @Samy-33

File: FusionIIIT/applications/academic_information/views.py [hunk 1]
```diff
@@ -1,3 +1,397 @@
...
+    return HttpResponse("Data Deleted SuccessFully")
+
+
+def add_course(request):
+    if request.method == "POST":
+        try:
+            c = Student.objects.get(id=request.POST['roll'])
+        except:
+            return HttpResponse("Student Does Not Exist")
+        if(request.POST['c1']):
```
Comment (Samy-33) [inline FusionIIIT/applications/academic_information/views.py]: Remove the parenthesis in if statements.

File: FusionIIIT/applications/academic_information/views.py [hunk 2]
```diff
@@ -1,3 +1,397 @@
...
+            return HttpResponse("Data Already Exists")
+        grades = Grades.objects.all()
+        context = {
+            'grades': grades,
+        }
+    return render(request, "ais/ais.html", context)
+
+
+def delete_grade(request):
+    print(request.POST['delete'])
```
Comment (Samy-33) [inline FusionIIIT/applications/academic_information/views.py]: remove `print`

File: FusionIIIT/applications/academic_information/views.py [hunk 3]
```diff
@@ -1,3 +1,397 @@
...
+        if(Student.objects.get(id=request.POST['delete'])):
+            Student.objects.get(id=request.POST['delete']).delete()
+        else:
+            return HttpResponse("Id Does not exist")
+    return HttpResponse("Data Deleted Successfully")
+
+
+def delete_advanced_profile(request):
+    if request.method == "POST":
+        print(request.POST['delete'])
```
Comment (Samy-33) [inline FusionIIIT/applications/academic_information/views.py]: remove `print`

**Pull request closed without merging**